stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/nativeimpl/InitWebSubSubscriberServiceEndpoint.java

```diff
@@ -94,7 +96,7 @@ public void execute(Context context) {
                                                      + stringTopicIdentifier);
```

Comment:
Regarding line 91 
Shall we use a variable for (BMap<String, BMap<String, BString>>) config.getRefField(4) to avoid redundant cast?

Reply:
Fixed with 62b3504